hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -198,24 +199,25 @@ public static HoodieTableMetaClient init(String basePath, HoodieTableType tableT
   }
 
   /**
-   * @param storageConf storage configuration.
+   * @param conf storage configuration.
    * @param basePath    base path of the Hudi table.
    * @return a new {@link HoodieTableMetaClient} instance.
    */
-  public static HoodieTableMetaClient createMetaClient(StorageConfiguration<?> storageConf,
+  public static HoodieTableMetaClient createMetaClient(Configuration conf,
                                                        String basePath) {
     return HoodieTableMetaClient.builder()
-        .setConf(storageConf).setBasePath(basePath).build();
+        .setConf(HoodieStorageUtils.getStorageConf(conf)).setBasePath(basePath).build();
```

Comment:
This needs to use `HadoopFSUtils.getStorageConfWithCopy(conf)` to be consistent with previous behavior.

Reply:
good catch!